thorlcr/activities/indexread/thindexread.cpp

```diff
@@ -37,6 +38,8 @@ class CIndexReadBase : public CMasterActivity
     bool partitionKey = false;
     StringBuffer fileName;
     unsigned fileStatsTableStart = NotFound;
+    size32_t configSequentialIOSize{unknownConfigValue};
+    size32_t configRandomIOSize{unknownConfigValue};
```

Comment:
there is/will be no config settings, so these members should be removed.
See othre comment, probably best if you cherry-pick (as 1st commit) the commit from https://github.com/hpcc-systems/HPCC-Platform/pull/20400 , and base changes from there.